app/otr/client/rocketchat.otr.js

```diff
@@ -69,6 +69,10 @@ Meteor.startup(function () {
 	});
 
 	onClientMessageReceived.use(function (message) {
+		if (message.t !== 'otr' || message.t !== 'otr-ack' || !message.notification) {
```

Comment:
remove this condition to OTR run great.

Reply:
This condition is required for system messages to work, as without this, it will try to decrypt the system message and run into error. We want decrypt functionality running for only otr messages.